ambari-logsearch/ambari-logsearch-web/src/app/components/service-logs-table/service-logs-table.component.less

```diff
@@ -110,6 +110,10 @@
     background: none transparent;
   }
 
+  table thead th {
```

Comment:
better be merged into the upper `table` selector block (the same for `table td` below as well)